cmd/simplai/bot.go

```diff
@@ -173,7 +189,7 @@ func run(
 			node.NewSubchainWithName(
 				prompt.SearxSearchInfo.Name,
 				updateSpinner(spinner, "searching"),
-				prompt.NewSearxSearch("https://search.inframonde.me"),
+				prompt.NewSearxSearch(searxURL),
```

Comment:
can you also fix the viper.GetString("searxurl") to be "searx-url" there's a typo there